File: warrior-legends-server/src/main/java/com/cream/warriorLegends/game/base/Role.java
package com.cream.warriorLegends.game.base;

import com.cream.warriorLegends.obj.common.position.Xy;
import lombok.Getter;
import lombok.Setter;


@Getter
@Setter
public class Role {
    private long id;
    private String nickName;
    private int level;
    private int hp;
    private int mp;
    private int mapId;
    private Xy xy;
    private long loginTime;
    private long logoutTime;
}

File: warrior-legends-cli/src/renderer/src/interface/Role.ts
import Xy from "@/interface/Xy";

interface Role {
  id: string;
  nickName: string;
  level: number;
  hp: number;
  mp: number;
  mapId: number;
  xy: Xy;
  loginTime: string;
  logoutTime: string;
  x: number;
}

export default Role;
